evap/evaluation/tests/test_models.py

```diff
@@ -363,6 +367,26 @@ def test_inactive_users_shown(self):
         self.assertIn(active_user, user_list)
         self.assertIn(inactive_user, user_list)
 
+    def test_can_be_marked_inactive_by_manager(self):
+        user = mommy.make(UserProfile)
+        mommy.make(Evaluation, participants=[user], state="new")
```

Comment:
there should be an assert where `can_be_marked_inactive_by_manager` is `True` in the beginning before adding the user as an participant, to make sure a different outcome is possible :)